validator/server_arb/validator_spawner.go

```diff
@@ -63,96 +45,25 @@ func DefaultArbitratorSpawnerConfigFetcher() *ArbitratorSpawnerConfig {
 	return &DefaultArbitratorSpawnerConfig
 }
 
-type JitSpawnerConfig struct {
-	ConcurrentRuns int  `koanf:"concurrent-runs-limit" reload:"hot"`
-	Cranelift      bool `koanf:"cranelift"`
-}
-
-type JitSpawnerConfigFecher func() *JitSpawnerConfig
-
-var DefaultJitSpawnerConfig = JitSpawnerConfig{
-	ConcurrentRuns: 0,
-	Cranelift:      true,
-}
-
-func JitSpawnerConfigAddOptions(prefix string, f *flag.FlagSet) {
-	f.Int(prefix+".concurrent-runs-limit", DefaultJitSpawnerConfig.ConcurrentRuns, "number of cuncurrent runs")
-	f.Bool(prefix+".cranelift", DefaultJitSpawnerConfig.Cranelift, "use Cranelift instead of LLVM when validating blocks using the jit-accelerated block validator")
-}
-
-// joint for comfort only - the two configs are entirely separate.
-type ValidationConfig struct {
-	Arbitrator ArbitratorSpawnerConfig `koanf:"arbitrator" reload:"hot"`
-	Jit        JitSpawnerConfig        `koanf:"jit" reload:"hot"`
-}
-
-var DefaultValidationConfig = ValidationConfig{
-	Jit:        DefaultJitSpawnerConfig,
-	Arbitrator: DefaultArbitratorSpawnerConfig,
-}
-
-func ValidationConfigAddOptions(prefix string, f *flag.FlagSet) {
-	ArbitratorSpawnerConfigAddOptions(prefix+".arbitrator", f)
-	JitSpawnerConfigAddOptions(prefix+".jit", f)
-}
-
 type ArbitratorSpawner struct {
 	stopwaiter.StopWaiter
 	count         int32
-	locator       *MachineLocator
+	locator       *server_common.MachineLocator
 	machineLoader *ArbMachineLoader
 	config        ArbitratorSpawnerConfigFecher
 }
 
-type readyMarker struct {
-	chanReady chan struct{}
-	boolReady int32
-	err       error
-}
-
 type valRun struct {
```

Comment:
Should this use `server_common.ValRun` instead?

Reply:
done in pt. 6